cos-gpu-installer-docker/entrypoint.sh

```diff
@@ -99,6 +102,11 @@ load_etc_os_release() {
   info "Running on COS build id ${BUILD_ID}"
 }
 
+fetch_cos_internal_gcs_bucket() {
```

Comment:
I don't think 'fetch' is the right verb for this, since data isn't being downloaded from anywhere. Would it make sense to call this 'set_cos_internal_gcs_bucket' instead?

Reply:
Done.